makefile

```diff
@@ -165,17 +166,26 @@ recipes/%/web:
 > @echo Recipe $* is not compatible with the web browser backend
 > exit 1
 
+# Check if index.js points to another recipe.
+%-indexCheck:
+> @if ! grep "require(\"../../../output/$*.Main/index.js\").main();" $(call webJs,$*) --quiet
+> then
+>   echo Error: $(call webJs,$*) points to another recipe:
+>   cat $(call webJs,$*)
+>   exit 1
+> fi
+
 .PHONY: %-web
 # Launches recipe in web browser
-%-web: $(call recipeDir,%) $(call webDir,%) %-build
+%-web: $(call recipeDir,%) $(call webDir,%) %-indexCheck %-build
 > @echo === Launching $* in the web browser ===
 > parcel $(call webHtml,$*) --out-dir $(call webDistDir,$*) --open
 
 .PHONY: %-buildWeb
```

Comment:
I'm not sure if phony targets work with pattern rules. Would be nice to find some documentation on this, although even if they aren't supported (or ignored) they don't seem to be hurting anything.